gazelle/js/tests/tsconfig_deps/types/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "types": ["jquery", "@testing-library/jest-dom", "ignored", "@test/ignored"]
```

Comment:
Why drop the ignored ones?

Reply:
I think it was originally just a mistake on my part.

Looking at it again, I'm not sure if these should be silently ignored or if they should fail. They're essentially unknown dependencies, right?

Reply:
Yeah, basically unknown dependencies.

I'd prefer keeping them to ensure we aren't changing existing behaviour. We can debate if it should error or silently ignore another time.